flask/app.py

```diff
@@ -1,6 +1,34 @@
-from flask import Flask
+from flask import Flask, render_template
+from firebase import firebase
+import os
+from flask.ext.wtf import Form
+from wtforms.fields import StringField, BooleanField
+from wtforms.validators import DataRequired
+from flask_wtf.csrf import CSRFProtect
+
 app = Flask(__name__)
+app.secret_key = os.environ.get('FIREBASE_KEY', '/.env')
```

Comment:
Yep so this line shouldn't stay like this, but I was having trouble figuring out the whole environment thing so I wrote this for now????

Reply:
We don't need the '/.env' parameter if we use environment variables (i.e. `export FIREBASE_KEY="secret_key"`)